QuizApp/QuizApp/Client/Account/AccountClient.swift

```diff
@@ -0,0 +1,54 @@
+import Foundation
+
+protocol AccountClientProtocol {
+
+    var data: AccountResponseClientModel { get async throws }
+
+    func updateData(name: String) async throws -> AccountResponseClientModel
+
+}
+
+private struct AccountRequestClientModel: Codable {
+
+    let name: String
+
+}
+
+class AccountClient: AccountClientProtocol {
+
+    private let path = "/v1/account"
+    private let securityStorage: SecurityStorageProtocol
+    private let networkClient: NetworkClientProtocol
+
+    var data: AccountResponseClientModel {
+        get async throws {
+            guard let token = securityStorage.accessToken else {
```

Comment:
Since you have created NetworkClient to manage common logic, which is excellent, why don't you move the fetching token and assign it to a request there?